theano/sandbox/cuda/basic_ops.py

```diff
@@ -578,26 +577,6 @@ def __init__(self, reduce_mask, scalar_op, pre_scalar_op=None):
         if pre_scalar_op:
             assert pre_scalar_op.nin == 1
 
-    def __eq__(self, other):
```

Comment:
Keep \__str__ the default string isn't the same and I prefer the one you removed.